index.html

```diff
@@ -8,9 +8,9 @@
     <link rel="stylesheet" href="style.css">
 </head>
 <body>
-    <div class="container" style="background-image:url('/home/ignatius/Documents/Microverse Projects/Portfolio/images/background-image.png');">
+    <div class="container">
         <div class="toolbar">
-            <p class="heading-1" style="color:blue ;">My logo</p>
+            <p class="heading">My logo</p>
```

Comment:
- [ ] Kindly as suggested by the previous reviewer, I think it would be better to add an <a> on your logo as adding an <a> tag to a logo allows users to click on the logo and be directed to the homepage of the website. 👍
This is a common convention in web design and helps users navigate easily to the main page of a website.
Kindly read this [article](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for a better understanding 😎😎